src/data/thunks.js

```diff
@@ -98,6 +97,32 @@ export function getProctoringSettings() {
   };
 }
 
+// todo: should this happen on its own or within updateAttemptAfter or somewhere else?
+// todo: need to update exam state to include exam access ticket for 1599 or later work?
+export function getExamAccessTicket() {
+  return async (dispatch, getState) => {
+    const { exam } = getState().examState;
+    if (!exam.id) {
+      logError('Failed to get exam settings. No exam id.');
+      handleAPIError(
+        { message: 'Failed to fetch exam access ticket. No exam id was found.' },
+        dispatch,
+      );
+      // return;
+    }
+    // try {
+    //   const examAccessTicketResponse = await fetchExamAccessTicket(exam.id);
```

Comment:
Note: so here, I'm fetching an exam ticket then grabbing the actual cookie.

Reply:
So with cookies there's no need for the frontend code to manage them. Because the backend code we wrote returns a set-cookie response the browser will do everything for us as far as storing it, handling expiration, and sending along to any request to *.edx.org.